src/app/users/users-profile/users-profile.component.html

```diff
@@ -1,53 +1,73 @@
-<h2 i18n>Profile</h2>
-<div mat-card-avatar *ngIf="userDetail._attachments; else noAttachment">
-    <img class="user-profile" [src]="imageSrc">
+<mat-toolbar>
+  <button mat-icon-button (click)="goBack()"><mat-icon>arrow_back</mat-icon></button>
+  <span i18n>Users</span>
+  <span class="toolbar-fill"></span>
+</mat-toolbar>
+
+<div class="space-container">
+  <mat-toolbar class="primary-color font-size-1">
+    <span i18n>{{userDetail.name}}</span>
+    <span class="toolbar-fill"></span>
+    <button mat-button class="margin-lr-3" *ngIf="(name === urlName) || (roles.indexOf('_admin') > -1)" i18n [routerLink]="['/users/update/', urlName]"><mat-icon>mode_edit</mat-icon><span>Edit Profile</span></button>
+    <button mat-button class="margin-lr-3" *ngIf="name === urlName" i18n (click)="changePasswordForm(userDetail)"><span>Change password</span></button>
+  </mat-toolbar>
+  <div class="view-container">
+    <mat-grid-list cols="3">
+      <mat-grid-tile>
+        <ng-container *ngIf="userDetail._attachments; else noAttachment">
+          <img class="user-profile" [src]="imageSrc">
+        </ng-container>
+        <ng-template #noAttachment>
+          <img class="user-profile" src="../assets/image.png">
+        </ng-template>
+      </mat-grid-tile>
+      <mat-grid-tile>
+        <mat-list>
+          <mat-list-item>
+            <h3 matLine i18n>Full Name</h3>
+            <h2 mat-line i18n><b>{{userDetail.firstName}} {{userDetail.middleName}} {{userDetail.lastName}}</b></h2>
```

Comment:
I think i18n is not needed in dynamic section , could you please remove it and check it another section too ?